app/controllers/api/stories_controller.rb

```diff
@@ -86,13 +86,16 @@ def filtered(resources)
   def included(relation)
     relation.includes(
       { audio_versions: [:audio_files] },
```

Comment:
could also include :audio_version_templates for distribution, and :distributions for series?

Reply:
hmm, I could add the distro av templates, those are included by default in the embedded distro, but the min version of series embedded in a story does not include distributions.

Reply:
it links to distributions, no? (not embedding). does the `count` matter?
```ruby 
# app/representers/api/min/series_representer.rb

  link :distributions do
    {
      href: api_series_distributions_path(represented),
      count: represented.distributions.count
    } if represented.id
  end
```

Reply:
also, this is the story distribution, not the series distro, so there are no templates associated with it.